templates/vhost/vhost_ssl_settings.erb

```diff
@@ -15,7 +15,9 @@
   ssl_buffer_size           <%= @ssl_buffer_size %>;
 <% end -%>
   ssl_protocols             <%= @ssl_protocols %>;
+<% if @ssl_ciphers != '' -%>
```

Comment:
I think it's safe to ignore my pull request because I think a more comprehensive solution is needed. I will try to come up with an idea and let you know.